### ISSUE TITLE
UI: Skills that need attention 

### ISSUE THREAD

**Issue by alexseyk** [ToranSharma/Duo-Strength#154]
After hidden skills are revealed the only way to hide them again is to reload the page 
Suggestions : 
1. place "reveal hidden skills " button on the top i.e. above the shown skills 
2. Change / toggle  it's state to "Hide skills"

i.e. make it work like "focus mode" button

**Comment by ToranSharma** [comment]
I suppose the reason I didn't add a button to hide them again is I didn't see the point. Why did the user click the reveal button if they wanted them gone again right after. I saw the button to reveal them as something that was for the odd occasion that they wanted to do something with the tree that wasn't one of the things that needs attention. I suppose I thought the feature would be used all the time, and not as something that would be switched on and off much.

That said I can add another button, or not remove the button after clicking but make it toggle the hiding

**Comment by alexseyk** [comment]
I agree that for a golden L5 tree makes little sense to unhide  and hide back  all the skills. However for an unfinished tree, after reading some tips(theory)  and  practice on castles,  i found myself wanting to go back to uncluttered view. Also took me some time to figure out that suggest skills was set to "random" and not to "first" , which are fine for completed & un-finishsed trees respectively.

**Comment by alexseyk** [comment]
Please add more customization options to "Show Only the Skills that Need Attention" , i.e. checkboxes to 
- Hide Cracked Skills 
- Hide Need Strengthening Skills 
- Hide Suggested Skill

Thus it should be a minimalistic google like view :)
